src/Action/RetrieveAutocompleteItemsAction.php

```diff
@@ -186,10 +186,6 @@ private function retrieveFilterFieldDescription(
     ): FieldDescriptionInterface {
         $fieldDescription = $admin->getFilterFieldDescription($field);
 
-        if (!$fieldDescription) {
-            throw new \RuntimeException(sprintf('The field "%s" does not exist.', $field));
```

Comment:
There are two ways to handle this. On master getFilterFieldDescription can only return the field description or an exception. 

1. Remove this exception because it will be triggered by the getFilterFieldDescription (that is the one I followed
2. Keep this line but calling the hasFilterFieldDescription first

Maybe the second is better, we should not rely on throwing logic exceptions. Code looks more robust with second one. WDYT?

I think I will make this change and wait for comments.

Reply:
I think the 2nd option is consistent with previous API narrowing PRs we have.

Reply:
Done

Reply:
Indeed, if we want to throw a Runtime exception instead.